File: LibWarDB2/Structures/WDB4Flags.cs
using System;

namespace LibWarDB2.Structures
{
    [Flags]
    public enum WDB4Flags : ushort
    {
        None = 0x00,
        HasOffsetMap = 0x01,
        /// <summary>
        /// This may be 'secondary keys' and is unrelated to WDC1+ relationships
        /// </summary>
        HasRelationshipData = 0x02,
        HasNonInlineIds = 0x04,
        /// <summary>
        /// // WDC1+
        /// </summary>
        IsBitpacked = 0x10
    }
}

File: LibWarDB2VB/Structures/WDB4Flags.vb
Imports System

Namespace LibWarDB2.Structures
    <Flags>
    Public Enum WDB4Flags ':ushort???
        None = &H0
        HasOffsetMap = &H1
        HasRelationshipData = &H2
        HasNonInlineIds = &H4
        IsBitpacked = &H10
    End Enum
End Namespace
